submission/submission-config/src/main/kotlin/ac/uk/ebi/biostd/common/properties/ClusterProperties.kt

```diff
@@ -11,6 +11,24 @@ data class ClusterProperties(
     val enabled: Boolean,
     val user: String,
     val key: String,
-    val server: String,
+    val lsfServer: String,
+    val slurmServer: String,
     val logsPath: String,
+    val default: Cluster,
```

Comment:
Following my previous comment, this property should be named something `type` or `mode` since this is decided at config level rather than runtime

Reply:
sure I will rename I just think transition will not be a move and forget and even team is still migrated resources from cluster and fixing issues.